conf/appsettings.json

```diff
@@ -39,7 +39,7 @@
 		// {outputname}: filename of output json file
 		"WindowsInteractiveCommandx86": "\"C:\\Program Files (x86)\\Windows Kits\\10\\Debuggers\\x64\\cdb.exe\" -z {dumppath}",
 		"WindowsInteractiveCommandx64": "\"C:\\Program Files (x86)\\Windows Kits\\10\\Debuggers\\x64\\cdb.exe\" -z {dumppath}",
-		"LinuxAnalysisCommand": "docker run -i --rm -v {dumpdir}:/dump linux-superdump /root/prepare_crash_artifacts_superdump.sh {dumpname} /dump/{outputname}",
+		"LinuxAnalysisCommand": "bash -c dotnet\\ /<path-to-superdump>/CoreDumpAnalysis/bin/Release/PublishOutput/CoreDumpAnalysis.dll\\ {dumpname}\\ {outputname}",
```

Comment:
<path-to-superdump> has to be replaced by a path that is accessible from inside bash, i.e. /mnt/c/...